tests/resolvers/ChatMessage/replyTo.spec.ts

```diff
@@ -0,0 +1,95 @@
+import "dotenv/config";
+import { replyTo as replyToResolver } from "../../../src/resolvers/ChatMessage/replyTo";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { ChatMessage } from "../../../src/models";
+import { beforeAll, afterAll, describe, it, expect, vi } from "vitest";
+import type { TestChatMessageType } from "../../helpers/chat";
+import {
+  createTestChatMessageWithoutReply,
+  createTestChatMessage,
+} from "../../helpers/chat";
+import { Types } from "mongoose";
+import { MESSAGE_NOT_FOUND_ERROR } from "../../../src/constants";
+
+let testChatMessage: TestChatMessageType;
+let testChatMessageWithoutReply: TestChatMessageType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
```

Comment:
The `beforeAll` block asynchronously sets up data. However, if one of the await calls (e.g., `createTestChatMessage`) fails, you won’t know which one because the error handling isn’t granular. Use try-catch around each await call for better error handling and debugging.

**Example**
```javascript
  try {
    MONGOOSE_INSTANCE = await connect();
  } catch (error) {
    console.error("Failed to connect to the database", error);
    throw error;
  }

  try {
    const temp = await createTestChatMessageWithoutReply();
    testChatMessageWithoutReply = temp[3];
  } catch (error) {
    console.error("Failed to create test chat message without reply", error);
    throw error;
  }
```